qiskit_machine_learning/connectors/torch_connector.py

```diff
@@ -0,0 +1,101 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A connector to use Qiskit (Quantum) Neural Networks as PyTorch modules."""
+
+import numpy as np
+
+from torch import Tensor
```

Comment:
I would assume, that since torch is not included conditionally, and must exist, that we will not by default be importing this class elsewhere? We just need to ensure that things work as expected when torch is not installed and things do not fail completely to load up etc.

Is there an expectation to convert the qgan pytorch discriminator to somehow using this connector? But my reading is that this is more the other way around in that whatever networks are provided here can be used more directly inside classical Torch code.

Reply:
added logic to check whether torch has been installed.